Makefile

```diff
@@ -59,6 +60,12 @@ generate-deploy: $(KUSTOMIZE) manifests
 generate-test: $(KUSTOMIZE) manifests
 	$(KUSTOMIZE) build config/test > config/test/kubemacpool.yaml
 
+generate-external: $(KUSTOMIZE) manifests
+	cp -r config/test config/external
+	cd config/external && \
```

Comment:
Do we return to the last directory we were?

Reply:
Yep, shell snippets are Make are very local, in fact they don't change dirs per line that's why we have to do  `&&`